event/models.py

```diff
@@ -8,8 +10,31 @@ def __str__(self) -> str:
         return self.name
 
 
+class EventLocation(models.Model):
+    name = models.CharField(max_length=150, blank=False, null=False)
+    # it is nullable as the location may be not always findable
+    google_maps_location = models.URLField(
+        max_length=500, blank=True, null=True
+    )
+
+    def __str__(self):
+        return self.name
+
+
+class HotTopic(models.Model):
+    name = models.CharField(max_length=100, primary_key=True)
+
+    def __str__(self):
+        return self.name
+
+
 # Create your models here.
 class Event(models.Model):
+    slug = models.SlugField(
+        max_length=350,
+        primary_key=True,
+        editable=False,
+    )
     title = models.CharField(max_length=400, blank=False, null=False)
     start_date = models.DateField(blank=False, null=False)
```

Comment:
date validation that end_date is after start_date
registration_deadline fall before start_date